src/Serilog.Settings.AppSettings/Serilog.Settings.AppSettings.csproj

```diff
@@ -0,0 +1,38 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>XML configuration (System.Configuration &lt;appSettings&gt;) support for Serilog.</Description>
+    <VersionPrefix>2.1.3</VersionPrefix>
+    <Authors>Serilog Contributors</Authors>
+    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
+    <AssemblyName>Serilog.Settings.AppSettings</AssemblyName>
+    <AssemblyOriginatorKeyFile>../../assets/Serilog.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    <PackageId>Serilog.Settings.AppSettings</PackageId>
+    <PackageTags>serilog;xml</PackageTags>
+    <PackageIconUrl>http://serilog.net/images/serilog-configuration-nuget.png</PackageIconUrl>
```

Comment:
It would be good to use only https urls in here so that when displaying package information in https://nuget.org , no "mixed content" warning show up

Reply:
OK. This was from the original version. I will update